package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-authentication-api-debugger-extension",
-  "version": "2.0.2",
+  "version": "2.1.0",
```

Comment:
@asmclean  shouln't it be a patch version? Now all customers have to explicitly update the version of the extension where with a patch that isn;t needed.